packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -195,7 +195,7 @@ export interface IContextualMenuProps extends React.Props<ContextualMenu>, IWith
   /**
    * Custom styling for the contextual menu.
    */
-  style?: IContextualMenuStyles;
+  styles?: IContextualMenuStyles;
```

Comment:
@dzearing Removing this prop would be a breaking change, but with the change to try to only rely on classnames I am no longer using it. I could remove it, show it as deprecated or just for this case keep on using it?